src/redux/slices/edit-suggestion.slice.ts

```diff
@@ -0,0 +1,67 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit"
+import { NewSuggestion } from "@/interfaces/new-suggestion"
+import { Area } from "@/constants/area"
+
+export interface EditSuggestionSlice {
+  activeArea: Area
+  lastEditedUuid?: string
```

Comment:
Perhaps just name this uuid since you're not really dealing with any other uuids in here